articles/tech-resources/deep-links-self-serve.md

```diff
@@ -0,0 +1,189 @@
+
+### Scenario 1: Universal Links Without Click Tracking
+
+Pros:
+- Very simple
+ 
+Cons:
+- No click tracking on univeral or Android App links
+
+This is the simplest option: if you disable SparkPost's click tracking for the mobile deep links in your email, they work just fine with your app with no further effort. This works because SparkPost does not alter untracked links in your email.
+
+#### Disabling Click Tracking
+- If you're using the SparkPost REST API, you can use the `data-msys-clicktrack` attribute to [disable click tracking for a single link in your email](https://developers.sparkpost.com/api/substitutions-reference.html#header-per-link-disabling-of-click-tracking):
+
+    ```html
+    <a data-msys-clicktrack="0" href="http://my.universallink.example.com/path/">Open in app</a>
+    ```
+
+- You can also [disable click tracking for a whole transmission request](https://developers.sparkpost.com/api/transmissions.html#header-options-attributes) by setting the `click_tracking` option to `false`.
+- [The SMTP API supports a similar flag](https://developers.sparkpost.com/api/smtp-api.html#header-open-and-click-tracking).
+- You can control [click tracking for SMTP messages](https://app.sparkpost.com/account/smtp) at the account level too.
+
+### Scenario 2: Deep Links With Click Tracking
+
+Pros:
+- Minimal development effort
+
+Cons:
+- Requires iOS/Android app support
+
+If you'd like to use SparkPost to track when people click the universal links in your email, you can use a custom tracking domain and have your app pass each click event to SparkPost. In this scenario, you nominate a link in your email as a mobile deep link by enabling click tracking on it and setting a specific "custom link sub-path". SparkPost replaces the link with a tracked version using your tracking domain and including the named path in the tracked URL. When your recipients click on the link, the domain and path match your published deep link and the mobile device opens your app. Your app then makes an HTTP request to the tracked URL which lets SparkPost know your recipient has visited that universal link.
+
+The setup steps are as follows:
+
+1. Configure a CDN to host your custom tracking domain.  One example can be found [here](https://www.sparkpost.com/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost/).
+**Note:** The CDN needs to support file hosting such as AWS CloudFront.
+1. [Configure a custom tracking domain on your SparkPost account.](https://app.sparkpost.com/account/tracking-domains)
+1. [Use a "custom link sub-path" in your links.](https://developers.sparkpost.com/api/substitutions-reference.html#header-custom-link-sub-paths)
+1. Publish the `apple-app-site-associate` or `assetlinks.json` file for your app using both the tracking domain and link sub-path.  These should be hosted on your CDN under  **\<your-tracking-domain>\\.well-known\\**. Your apple-app-site-association file might include the following:
+    ```javascript
```

Comment:
I wonder if we include example deep link specs for Android.  We could just indent the 2 example code blocks by 4 spaces to have them appear under the numbered list item.